cypress/integration/login.spec.js

```diff
@@ -118,4 +117,6 @@ context('Login', () => {
                     .should('contain', '/lost-password');
             });
     });
+   
+
```

Comment:
Get rid of this whitespace.

But keep an empty line between `it`s in the spec files. It really helps readability.